pkgs/tools/security/yubihsm-connector/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, libusb, buildGo117Module, fetchFromGitHub, pkg-config }:
+
+buildGo117Module rec {
+  pname = "yubihsm-connector";
+  version = "3.0.2";
+
+  src = fetchFromGitHub {
+    owner = "Yubico";
+    repo = "yubihsm-connector";
+    rev = version;
+    sha256 = "sha256-FQ64tSZN55QpXjMZITzlWOPTKSgnoCpkRngQUQHVavc=";
+  };
+  vendorSha256 = "sha256-sKCZ8KUUwMQWGITuWNCDO9p+awN3JGi+xYzrGy8Qz3g=";
+
+  proxyVendor = true;
+
+  preBuild = "go generate";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ libusb ];
+
+  meta = with lib; {
+    homepage = "https://developers.yubico.com/yubihsm-connector/";
+    maintainers = with maintainers; [ matthewcroughan ];
+    license = licenses.apsl20;
```

```suggestion
    license = licenses.asl20;
```

`apsl20` is "Apple Public Source License 2.0"

---

Whoops!